CHANGE_NOTES.md

```diff
@@ -0,0 +1,10 @@
+The mapBasicAttrbuteType function now successfully processes arrays.
```

Comment:
Can you remove this file. It's not needed for the project

Reply:
Yes I can remove it